### DISCUSSION TITLE
Implicit multicursor isn't consistent, how do I turn off?

### DISCUSSION THREAD

**Discussion by CamdenClark** [zed-industries/zed#17518] [Feature Requests]
This is a huge problem for editing clojure code and it's really strange and hard to explain, so here's a video:

https://github.com/user-attachments/assets/351445bb-b296-4a55-95b2-026907b98ed1

I'm using vim mode. I press `ciw` (change inner word) and the word whose cursor i'm under is deleted, but then when I start to type, it applies the changes I type to the other instances in the file.

How do I just disable this feature? I don't care at all about the implicit multicursor, and would rather have it off than be buggy.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
